src/lib/types.ts

```diff
@@ -205,6 +205,98 @@ export interface RTPCreditPaymentInstruction extends PaymentInstruction {
   creditor: Party,
 }
 
+/**
+ * ACH Local Instrument Codes as defined in NACHA standards.
+ * These codes identify the specific type of ACH transaction.
+ */
+export const ACHLocalInstrumentCode = {
+  /** Corporate Credit or Debit */
+  CorporateCreditDebit: 'CCD',
+  /** Prearranged Payment and Deposit */
+  PrearrangedPaymentDeposit: 'PPD',
+  /** Internet-Initiated Entry */
+  InternetInitiated: 'WEB',
+  /** Telephone-Initiated Entry */
+  TelephoneInitiated: 'TEL',
+  /** Point-of-Purchase Entry */
+  PointOfPurchase: 'POP',
+  /** Accounts Receivable Entry */
+  AccountsReceivable: 'ARC',
+  /** Back Office Conversion */
+  BackOfficeConversion: 'BOC',
+  /** Represented Check Entry */
+  RepresentedCheck: 'RCK',
+} as const;
+
+export type ACHLocalInstrument = 
+  (typeof ACHLocalInstrumentCode)[keyof typeof ACHLocalInstrumentCode];
+
+export const ACHLocalInstrumentCodeDescriptionMap = {
+  'CCD': 'Corporate Credit or Debit',
+  'PPD': 'Prearranged Payment and Deposit',
+  'WEB': 'Internet-Initiated Entry',
+  'TEL': 'Telephone-Initiated Entry',
+  'POP': 'Point-of-Purchase Entry',
+  'ARC': 'Accounts Receivable Entry',
+  'BOC': 'Back Office Conversion',
+  'RCK': 'Represented Check Entry',
+} as const;
+
+/**
+ * ACH Service Level Codes as defined in ISO20022 standards.
+ * These codes identify the priority or service level of the payment.
+ */
+export const ACHServiceLevelCode = {
+  /** Normal Urgency */
+  NormalUrgency: 'NURG',
+  /** High Urgency */
+  HighUrgency: 'URGP',
+  /** Same Day */
+  SameDay: 'SDVA',
+} as const;
+
+export type ACHServiceLevel = 
+  (typeof ACHServiceLevelCode)[keyof typeof ACHServiceLevelCode];
+
+export const ACHServiceLevelCodeDescriptionMap = {
+  'NURG': 'Normal Urgency',
+  'URGP': 'High Urgency',
+  'SDVA': 'Same Day',
+} as const;
+
+/**
+ * ACH Instruction Priority Codes as defined in ISO20022 standards.
+ * These codes identify the priority of the instruction.
+ */
+export const ACHInstructionPriorityCode = {
+  /** Normal */
+  Normal: 'NORM',
+  /** High */
+  High: 'HIGH',
+} as const;
+
+export type ACHInstructionPriority = 
+  (typeof ACHInstructionPriorityCode)[keyof typeof ACHInstructionPriorityCode];
+
+export const ACHInstructionPriorityCodeDescriptionMap = {
+  'NORM': 'Normal',
+  'HIGH': 'High',
+} as const;
```

Comment:
These are all not necessary for now. Please use the standards 'NORM' and 'NURG' in 'serialize` here.

We can add the Enums at a later stage.